packages/eslint-plugin/src/rules/local-cx-xcss/__tests__/rule.test.ts

```diff
@@ -0,0 +1,94 @@
+import { typeScriptTester as tester } from '../../../test-utils';
+import { localCXXCSSRule } from '../index';
+
+tester.run('local-cx-xcss', localCXXCSSRule, {
+  valid: [
+    `
+    import { cx } from '@compiled/react';
+
+    <Component xcss={cx({})} />
+  `,
+    `
+    import { cx } from '@compiled/react';
+
+    <Component innerXcss={cx({})} />
+  `,
+    `
+    import { cx } from '@compiled/react';
+
+    const styles = styleMap({
+      text: { color: 'red' },
+      bg: { background: 'blue' },
+    });
+
+    <Button innerXcss={cx(styles.text, styles.bg)} />;
+  `,
+    `
+    // Ignore cx usage not from compiled
+    const styles = styleMap({
+      text: { color: 'red' },
+      bg: { background: 'blue' },
+    });
+
+    const joinedStyles = cx(styles.text, styles.bg);
+
+    <Button xcss={styles} />;
+    `,
+    `
+    // Ignore cx usage not from compiled
+      const styles = cx({});
```

Comment:
/nit make the spacing consistent, so this line and the next should be lined up with the commend above